modules/geo-location/class.jetpack-geo-location.php

```diff
@@ -249,7 +249,9 @@ public function register_rss_hooks() {
 	 * Add the georss namespace during RSS generation.
 	 */
 	public function rss_namespace() {
-		echo PHP_EOL . 'xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#"' . PHP_EOL;
+		echo PHP_EOL . "\t" . 'xmlns:georss="http://www.georss.org/georss\"';
```

Comment:
Is the trailing backslash needed?

Reply:
Nope, good catch! Fixed in 731198c39bd1cf5d3b96e9a7ac0303583e7a1af5